package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -459,7 +459,7 @@ class NCDFReader(base.ReaderBase):
     _Timestep = Timestep
 
     def __init__(self, filename, n_atoms=None, **kwargs):
-        self._mmap = kwargs.pop('mmap', False)
+        self._mmap = kwargs.pop('mmap', None)
```

Comment:
please be explicit about the `mmap` keyword in the function instead of poping from kwargs. This will also make it easier to spot the default value in an interpreter setting.

Reply:
added to commit